[2025-02-18T06:42:54.131+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T06:42:54.161+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.extract manual__2025-02-18T06:42:42.501223+00:00 [queued]>
[2025-02-18T06:42:54.182+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.extract manual__2025-02-18T06:42:42.501223+00:00 [queued]>
[2025-02-18T06:42:54.186+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-18T06:42:54.223+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract> on 2025-02-18 06:42:42.501223+00:00
[2025-02-18T06:42:54.244+0000] {standard_task_runner.py:72} INFO - Started process 3908 to run task
[2025-02-18T06:42:54.256+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'extract', 'manual__2025-02-18T06:42:42.501223+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/DAG-fraud.py', '--cfg-path', '/tmp/tmp_fllu868']
[2025-02-18T06:42:54.266+0000] {standard_task_runner.py:105} INFO - Job 161: Subtask extract
[2025-02-18T06:42:55.559+0000] {task_command.py:467} INFO - Running <TaskInstance: ml_pipeline.extract manual__2025-02-18T06:42:42.501223+00:00 [running]> on host 8538735272b1
[2025-02-18T06:42:56.078+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='group1' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T06:42:42.501223+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-18T06:42:42.501223+00:00'
[2025-02-18T06:42:56.105+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T06:42:56.321+0000] {base.py:84} INFO - Retrieving connection 'postgres_***'
[2025-02-18T06:42:56.444+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.9/site-packages/pandas/io/sql.py:761 UserWarning: pandas only support SQLAlchemy connectable(engine/connection) ordatabase string URI or sqlite3 DBAPI2 connectionother DBAPI2 objects are not tested, please consider using SQLAlchemy
[2025-02-18T06:42:58.188+0000] {DAG-fraud.py:42} INFO - Data extraction successful.
[2025-02-18T06:42:58.197+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T06:42:58.214+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T06:42:58.218+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=extract, run_id=manual__2025-02-18T06:42:42.501223+00:00, execution_date=20250218T064242, start_date=20250218T064254, end_date=20250218T064258
[2025-02-18T06:42:58.332+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T06:42:58.448+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-18T06:42:58.464+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
